websites/B/Board Game Arena/games/E/escapefromthehiddencastle.ts

```diff
@@ -0,0 +1,45 @@
+import { GamePresence } from "..";
+import {
+	getActivePlayerId,
+	getCurrentGameState,
+	getCurrentGameStateType,
+	getPlayerAvatar,
+	getPlayerData,
+	getPlayerScore,
+	getUserPlayerId,
+} from "../../util";
+
+const escapefromthehiddencastle: GamePresence = {
+	logo: "https://i.imgur.com/D7Y5Gcm.png",
```

Comment:
Is this the correct logo?

Reply:
I can confirm this is the correct logo. https://boardgamearena.com/gamepanel?game=escapefromthehiddencastle